### PR TITLE
Erin doc edits

### PR DESCRIPTION
**Pull request by ErinWeisbart**

### CODE REVIEW

**Comment by ErinWeisbart** [comment]
@bethac07 does this satisfy #10 ?

**Comment by ErinWeisbart** [comment]
(example-datasets.md still needs a little cleanup.)

**Comment by bethac07** [comment]
Indeed it does solve #10, thanks!

**Comment by ErinWeisbart** [comment]
@bethac07 missing license for *C. elegans* dataset and citation for Human PLP1 dataset. Can you help with those?

**Comment by bethac07** [comment]
I don't know what the license is for the worms, it's not on [BBBC](https://bbbc.broadinstitute.org/BBBC012). I don't know that was originally ever collected. Might be worth having Anne reach out.

Citation for PLP1 should nearly certainly be the below, but again, might be worth having Anne confirm, she made that dataset herself.

Lacoste J, Haghighi M, Haider S, Reno C, Lin Z, Segal D, Qian WW, Xiong X, Teelucksingh T, Miglietta E, Shafqat-Abbasi H, Ryder PV, Senft R, Cimini BA, Murray RR, Nyirakanani C, Hao T, McClain GG, Roth FP, Calderwood MA, Hill DE, Vidal M, Yi SS, Sahni N, Peng J, Gingras A, Singh S, Carpenter AE, Taipale M. Pervasive mislocalization of pathogenic coding variants underlying human disorders. Cell. 2024. doi: 10.1016/j.cell.2024.09.003

**Comment by AnneCarpenter** [comment]
Confirmed PLP2! Asked Javier just now by email

File: piximi-documentation/technical-faq.md [hunk 1]
```diff
@@ -49,22 +51,26 @@ You can also serve the application locally using Docker. The instructions to do
 
 No internet connection is necessary to save or load projects and models.
```
Comment (bethac07) [inline piximi-documentation/technical-faq.md:52]: @ErinWeisbart can you caveat this? You don't need an internet connection for _classifier_ models but you do for segmentation models (and for Cellpose you need to full on send them to the internet)

**Comment by AnneCarpenter** [comment]
Javier confirms: "please place those images under CC0."
Thanks!

File: piximi-documentation/technical-faq.md (hunk 1 above)
Comment (ErinWeisbart) [reply piximi-documentation/technical-faq.md]: done. (FYI I had edited the technical FAQ for formatting but not content. Happy to add any additional content edits you suggest)

**Comment by ErinWeisbart** [comment]
@bethac07 This is ready for review.

File: piximi-documentation/technical-faq.md (hunk 1 above)
Comment (bethac07) [reply piximi-documentation/technical-faq.md]: TY! That was the only content issue I noticed but will add others if I see them

**Review by bethac07 — APPROVED** [review]

**Pull request merged**